tasks/pipeline-instcode.yml

```diff
@@ -42,8 +42,6 @@
       dest: "/usr/lib/archivematica/MCPServer"
     - src: "{{ archivematica_src_dir }}/archivematica/src/MCPServer/share"
       dest: "/usr/share/archivematica/MCPServer"
-    - src: "{{ archivematica_src_dir }}/archivematica/src/MCPServer/share/mysql"
-      dest: "/usr/share/dbconfig-common/data/archivematica-mcp-server/install/mysql"
```

Comment:
Is this going to cause a problem for the pre-migrations case? I haven't seen any problems so far.